### PR TITLE
Rename some Peep member variables to use TitleCase

### PR DESCRIPTION
**Pull request by tupaschoal**
Continues #10653

It might be easier to review on a commit by commit basis

### CODE REVIEW

File: src/openrct2/peep/GuestPathfinding.cpp [hunk 1]
```diff
@@ -1646,9 +1646,9 @@ static int32_t guest_path_find_leaving_park(Peep* peep, uint8_t edges)
 static int32_t guest_path_find_park_entrance(Peep* peep, uint8_t edges)
 {
     // If entrance no longer exists, choose a new one
-    if ((peep->PeepFlags & PEEP_FLAGS_PARK_ENTRANCE_CHOSEN) && peep->current_ride >= gParkEntrances.size())
+    if ((peep->PeepFlags & PEEP_FLAGS_PARK_ENTRANCE_CHOSEN) && peep->CurrentRide >= gParkEntrances.size())
     {
-        peep->current_ride = 0xFF;
+        peep->CurrentRide = 0xFF;
```
Comment (Gymnasiast) [inline src/openrct2/peep/GuestPathfinding.cpp:1651]:
```suggestion
        peep->CurrentRide = RIDE_ID_NULL;
```

File: src/openrct2/peep/Peep.h [hunk 2]
```diff
@@ -632,36 +632,36 @@ struct Peep : SpriteBase
...
-    uint8_t photo2_ride_ref;
-    uint8_t photo3_ride_ref;
-    uint8_t photo4_ride_ref;
-    uint8_t current_ride;
-    StationIndex current_ride_station;
-    uint8_t current_train;
+    uint8_t Photo2RideRef;
+    uint8_t Photo3RideRef;
+    uint8_t Photo4RideRef;
+    uint8_t CurrentRide;
```
Comment (Gymnasiast) [inline src/openrct2/peep/Peep.h:638]:
```suggestion
    ride_id_t CurrentRide;
```

File: src/openrct2/peep/Peep.cpp [hunk 3]
```diff
@@ -1860,18 +1860,18 @@ void Peep::FormatActionTo(void* argsV) const
...
         case PEEP_STATE_WATCHING:
-            if (current_ride != RIDE_ID_NULL)
+            if (CurrentRide != RIDE_ID_NULL)
             {
-                auto ride = get_ride(current_ride);
+                auto ride = get_ride(CurrentRide);
                 if (ride != nullptr)
                 {
-                    ft.Add<rct_string_id>((current_seat & 0x1) ? STR_WATCHING_CONSTRUCTION_OF : STR_WATCHING_RIDE);
+                    ft.Add<rct_string_id>((CurrentSeat & 0x1) ? STR_WATCHING_CONSTRUCTION_OF : STR_WATCHING_RIDE);
```
Comment (Gymnasiast) [inline src/openrct2/peep/Peep.cpp:1868]: Should be `standing_flags` (which is in a union with `CurrentSeat`)

File: src/openrct2/peep/Peep.cpp [hunk 4]
```diff
@@ -1860,18 +1860,18 @@ void Peep::FormatActionTo(void* argsV) const
...
                     ride->FormatNameTo(ft.Buf());
                 }
             }
             else
             {
-                ft.Add<rct_string_id>((current_seat & 0x1) ? STR_WATCHING_NEW_RIDE_BEING_CONSTRUCTED : STR_LOOKING_AT_SCENERY);
+                ft.Add<rct_string_id>((CurrentSeat & 0x1) ? STR_WATCHING_NEW_RIDE_BEING_CONSTRUCTED : STR_LOOKING_AT_SCENERY);
```
Comment (Gymnasiast) [inline src/openrct2/peep/Peep.cpp:1874]: Should also be `standing_flags`.

File: src/openrct2/peep/Peep.cpp (hunk 3 above)
Comment (tupaschoal) [reply src/openrct2/peep/Peep.cpp:1868]: Nice catch!

**Comment by tupaschoal** [comment]
I've dealt with the issues, will merge when CIs finish

File: src/openrct2/peep/Peep.h [hunk 5]
```diff
@@ -632,36 +632,36 @@ struct Peep : SpriteBase
...
-    uint8_t photo2_ride_ref;
-    uint8_t photo3_ride_ref;
-    uint8_t photo4_ride_ref;
-    uint8_t current_ride;
-    StationIndex current_ride_station;
-    uint8_t current_train;
+    uint8_t Photo2RideRef;
+    uint8_t Photo3RideRef;
+    uint8_t Photo4RideRef;
+    ride_id_t CurrentRide;
```
Comment (duncanspumpkin) [inline src/openrct2/peep/Peep.h:638]: You can't change the field size without recreating all the replays.

Comment (tupaschoal) [reply src/openrct2/peep/Peep.h:638]: Argh, right, let me revert it (ended up reverting the wrong patch

**Pull request merged**